### ISSUE TITLE
Location Issuses

### ISSUE THREAD

**Issue by hankalf** [ketilmo/balena-ads-b#141]
I am having issues with the services not wanting to start due to missing lat & long. I have entered the coordinates under the "Location" tab in balenaCloud as well as entered them under Device Variables. 

![Capture](https://github.com/user-attachments/assets/b95402f2-d934-4190-83b9-475de2e04bb5)

![image](https://github.com/user-attachments/assets/a606781d-cdd5-440d-83ff-7fd0a77be4d0)

I have looked through the "Issues & Discussions" and it does not appear that anyone else is having this issue. 

Is someone able to assist with on a fix? 

Thank you.

**Comment by shawaj** [comment]
@hankalf you need to follow the instructions here... https://github.com/ketilmo/balena-ads-b?tab=readme-ov-file#part-2--setup-balena-and-configure-the-device

**Comment by hankalf** [comment]
I followed that guide again, and after a full redeployed image, I was able to get that issue fixed. 

Now I am hit with a new issue on 3 different services. 

1. adsb-exchange - error connecting to service via terminal. 

![image](https://github.com/user-attachments/assets/8bac81e5-5c9f-4869-816f-af98d1d2122e)
![image](https://github.com/user-attachments/assets/0e45591b-bd1d-420f-97c1-90fabc97791e)



3. piaware - same error as adsb-exchange.

![image](https://github.com/user-attachments/assets/d701568a-f29c-4e2c-86f8-ca7ead3aa020)
![image](https://github.com/user-attachments/assets/78f29243-4faa-446a-ad19-c868557eb9e8)

5. planefinder - same issue. 

![image](https://github.com/user-attachments/assets/89fcf56f-b03f-4e1b-ad07-0f9b61baa56f)
![image](https://github.com/user-attachments/assets/e32021c1-7d36-4be9-b18e-892aa6adda7c)

**Comment by shawaj** [comment]
This issue is related to #135 and the fix for this is in #137 but hasn't been merged yet. 

You can try manually enabling the containers by clicking the play button next to them but not sure exactly how long that will enable for.

**Comment by ketilmo** [comment]
Hi guys,

Thanks for following up on this, @shawaj. And sorry for the inconvenience, @hankalf. I will merge the fix as soon as I've had the chance to review it and update this ticket when it's done. You will have to do a new deployment of the code, @hankalf, either through the blue "Deploy to balena"-button or the `balena push` command line tool.

**Comment by ketilmo** [comment]
Hi @hankalf,

The fix has now been merged. Please try deploying the fresh code to your device, and let us know how it goes.

**Comment by hankalf** [comment]
Hi @ketilmo & @shawaj, 

Thank you both for your input. I really appreciate it! After redeploying the code to the device, I was able to get into the terminal with no issues. I have now successfully got everything up and running. 

I came across this build while troubleshooting too many issues with a raspberry pi radar feeder. This build is everything I was wanting in my project as I am deploying this at a secondary home that has a full view of the sky all around, so remotely monitoring was a must. I will definitely be following the project for any further updates or improvements! 


Thanks again!
